os-replace.sh

```diff
@@ -18,7 +28,10 @@ SSH_KEY=${TEMPDIR}/id_rsa
 ssh-keygen -f "${SSH_KEY}" -N "" -q -t rsa-sha2-256 -b 2048
 SSH_KEY_PUB="${SSH_KEY}.pub"
 
-INSTALL_CONTAINERFILE=${TEMPDIR}/Containerfile.install
+git submodule update
```

Comment:
This command might miss some updates depending on when this script is triggered.
I'd suggest to adopt the renovate way to update the git submodule.
We already implemented it with bootc repos when syncing between upstream to downstream.

Reply:
Yes, I do want to use renovate to update this submodule, I didn't find there is any explicit config being added to enable it in bootc, so I assume renovatebot  will detect the submodule and update it by default, please help to add the missing renovate config if needed to enable the auto-update, thanks.

Regarding to this command, it's because this repo/branch is checkout by tmt, and I found the submodule folder is empty when runtime code reach here, so I force to update the submodule here.

Reply:
Added a renovate global configuration to enable git-submodules renovation. Take a look at the Dependency Dashboard #105.
Re- the second point. I think there is a git configuration which updates recursively when doing a checkout. I'd suggest looking into it.